examples/cosmos-sdk/transfer.md

```diff
@@ -0,0 +1,243 @@
+# Atomkraft test project
+
+## Token transfer
+
+### Prerequisite
+
+- [`python`](https://www.python.org/downloads) 3.10: For Atomkraft
+- [`pip`](https://pip.pypa.io/en/stable/installation): To install Atomkraft executable
+  - Normally after installing Python, you may just want to execute `python -m ensurepip --upgrade` to have `pip` installed.
+- [`poetry`](https://python-poetry.org/docs/#installation): For Atomkraft project
+- [`git`](https://github.com/git-guides/install-git): To clone Atomkraft template and Cosmos-SDK chain repo
+- [`make`](https://www.gnu.org/software/make): Compile Cosmos-SDK chain binary
+- [`go`](https://go.dev/doc/install): Compile Cosmos-SDK chain binary
+- [`java`](https://www.java.com/download): For [Apalache](https://apalache.informal.systems) model checker
+
+#### `ubuntu`
+
+```
+apt update -y && apt upgrade -y
+apt install pip3 git golang curl default-jre -y
+pip install --upgrade poetry
+```
+
+#### `archlinux`
+
+```
+pacman -Syu python-pip python-poetry git make go jre-openjdk gcc --noconfirm --needed
+```
+
+### Install Atomkraft
+
+```
+pip install --upgrade atomkraft
+```
+
+### Initialize project
+
+```
+atomkraft init transfer
+cd transfer
+```
+
+### Turn on Poetry shell
+
+```
+poetry shell
+```
+
+### Create a model specification
+
+We will model a simple token transfer between two users.
+Alice and Bob both start with 100 tokens.
+At each step, Alice or Bob will send some amount of tokens to the other person.
+We will use TLA+ to specify this model.
+You can use the following code to _jump-start_ a new model at `models/transfer.tla`.
+
+```
+cat > models/transfer.tla << EOF
+---- MODULE transfer ----
+EXTENDS Apalache, Variants, Integers, FiniteSets
```

Comment:
I'm not sure that using Variants is a good idea for a first, simplified example. We don't need that the user is able to write TLA+ specs, but at least we want the user to be able to understand them. Also, in a later version of this tutorial we should add more comments to the spec.